bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableConstants.java

```diff
@@ -59,7 +59,7 @@
   /**
    * A User-Agent token to be added to User-Agent request header.
    */
-  public static final String USER_AGENT = "bigtable-hbase/" + getVersion();
+  public static final String USER_AGENT = getVersion();
```

Comment:
Can you update the code to getUserAgent() and update the code there to use the right property name?